contrib/lorri.nix

```diff
@@ -0,0 +1,38 @@
+{ config, systemd, ... }:
+
+let
+  lorri = import
+    (fetchTarball "https://github.com/target/lorri/tarball/rolling-release")
+    { };
+  socketUnit = "lorri";
+  socketPath = "lorri/daemon.socket";
```

Comment:
Let's not have these as variables, and instead just use "lorri" and "lorri/daemon.socket". They're not really variable, just trying to reduce a bit of duplication I think in a way that doesn't get us much.

Reply:
That's what flokli said :) https://github.com/NixOS/nixpkgs/pull/72889#discussion_r345109924